index.html

```diff
@@ -11,13 +11,76 @@
   <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Roboto">
 </head>
 
-<body>
-  <h1 class="title">Front-End Tools</h1>
-  <div class="tech">
-    <img src="assets/react_icon.svg" class="tech-logo" alt="A React logo">
-    <h3 class="tech-name">ReactJS</h3>
-    <h4 class="tech-relase">Relase: 2013</h4>
-    <p class="tech-desc">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+<body class="container">
+  <!-- Front-End Tools-->
+  <h1 class="title m-top-50 m-bottom-50">Front-End Tools</h1>
```

Comment:
I think this 'm-top-50' is unnecessary, is the same as adding it directly on the css and it makes the html less simple.

Reply:
I think the top can be handled by adding padding in the `.container` but i'd leave the bottom one because if we add it in the `title` everywhere we use the title we'll have a 50px margin and that should depend on the context of where it's placed.